cpp/ycm/ClangCompleter/CompletionData.cpp

```diff
@@ -152,7 +160,9 @@ std::string RemoveTwoConsecutiveUnderscores( std::string text ) {
 } // unnamed namespace
 
 
-CompletionData::CompletionData( const CXCompletionResult &completion_result ) {
+CompletionData::CompletionData( const CXCompletionResult &completion_result )
+    :original_string_()
```

Comment:
Adding `:original_string_()` isn't necessary. You can remove it and get the same behavior.